oauth2client/contrib/appengine.py

```diff
@@ -89,11 +89,11 @@ def _safe_html(s):
     return cgi.escape(s, quote=1).replace("'", '&#39;')
 
 
-class InvalidClientSecretsError(Exception):
-    """The client_secrets.json file is malformed or missing required fields."""
+class Error(Exception):
+    """Base error for this module."""
 
 
-class InvalidXsrfTokenError(Exception):
+class InvalidXsrfTokenError(Error):
```

Comment:
Alright, makes sense.  I'll go back and revise this PR to a generalized exception refactoring.  Then file the few other issues that came up, and the associated PRs.

If I do the PRs right away, should I still file the issues separately?